### PR TITLE
feat: Output table sizes while calculating

### PR DESCRIPTION
**Pull request by jdmulloy**
Adds table sizes to output while running checks to add transparency to the process.

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A SRE team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a SRE ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/spaces/SRE/pages/28967861/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Comment by jdmulloy** [comment]
Adds output like this while checking each server
```
Checking table sizes for stage-edx-shared-002-us-east-1c.ciqreuddjk02.us-east-1.rds.amazonaws.com
RDS Name                                           Database Name                  Table Name                                                            
stage-edx-shared-002-us-east-1c                    reports_2_0                    module_engagement                                                                 41,022.00 MB
stage-edx-shared-002-us-east-1c                    reports_dev                    answer_distribution                                                               16,722.00 MB
stage-edx-shared-002-us-east-1c                    reports_emr_upgrade            answer_distribution                                                               12,908.00 MB
stage-edx-shared-002-us-east-1c                    reports_1_0                    student_engagement_wide_daily                                                     11,566.00 MB
stage-edx-shared-002-us-east-1c                    reports_emr_upgrade            course_enrollment_birth_year_daily                                                11,566.00 MB
stage-edx-shared-002-us-east-1c                    reports_1_0                    student_engagement_daily                                                           7,970.00 MB
stage-edx-shared-002-us-east-1c                    reports_2_0                    video_timeline                                                                     6,061.00 MB
stage-edx-shared-002-us-east-1c                    reports_1_0                    answer_distribution                                                                3,277.00 MB
stage-edx-shared-002-us-east-1c                    reports_1_0                    course_enrollment_birth_year_daily                                                 2,490.00 MB
stage-edx-shared-002-us-east-1c                    reports_2_0                    course_enrollment_mode_daily                                                       1,913.05 MB
stage-edx-shared-002-us-east-1c                    reports_emr_upgrade            course_enrollment_education_level_daily                                            1,569.00 MB
stage-edx-shared-002-us-east-1c                    reports_2_0                    course_enrollment_daily                                                            1,501.08 MB```

File: util/jenkins/check_table_size/check_table_size.py [hunk 1]
```diff
@@ -76,6 +76,8 @@ def check_table_growth(rds_list, username, password, threshold, rds_threshold):
         table_list = []
         for db in rds_list:
             print("Checking table sizes for {}".format(db["Endpoint"]))
+            format_string_header = "{:<50} {:<30} {:<70}"
+            print(format_string_header.format("RDS Name","Database Name", "Table Name", "Size"))
```
Comment (adzuci) [inline util/jenkins/check_table_size/check_table_size.py:80]: nit: missing space after comma

File: util/jenkins/check_table_size/check_table_size.py [hunk 2]
```diff
@@ -102,12 +104,15 @@ def check_table_growth(rds_list, username, password, threshold, rds_threshold):
                 threshold_limit = threshold
             for tables in rds_result:
                 temp_dict = {}
-                if tables[2] is not None and tables[2] > float(threshold_limit):
+                if tables[2] is not None:
```
Comment (adzuci) [inline util/jenkins/check_table_size/check_table_size.py:107]: Why did we remove the and bit?

**Review by adzuci — APPROVED** [review]
LGTM

File: util/jenkins/check_table_size/check_table_size.py (hunk 2 above)
Comment (jmbowman) [reply util/jenkins/check_table_size/check_table_size.py:107]: That condition got moved further down, so we can display the size of tables that don't exceed the threshold.

File: util/jenkins/check_table_size/check_table_size.py (hunk 1 above)
Comment (jmbowman) [reply util/jenkins/check_table_size/check_table_size.py:80]: Looks like that was copied from the code below for displaying the tables over the threshold, which would ideally also be fixed.  Later, we should either add a linter here or move this code somewhere that already has linting set up.

**Review by jmbowman — APPROVED** [review]
Would be nice to fix the formatting nit, but should be ok to go as is.

**Pull request closed without merging**